### ISSUE TITLE
S operation concatenates global average- and max-pooling support features in depth, and halve dimensions by a convolutional layer to generate salient support features, then T tile a salient support features to make the tiled support features have a same spatial size as query features, so F can concatenate tiled support features to query features

### ISSUE THREAD

**Issue by God-song** [AICyberTeam/OSCD-datasets#3]
![image](https://user-images.githubusercontent.com/38659788/199442208-df108763-633e-4236-ba0d-3432ef0a1b32.png)
S operation concatenates global average- and max-pooling support features in depth, and halve dimensions by a [convolutional layer](https://www.sciencedirect.com/topics/computer-science/convolutional-layer) to generate salient support features, then T tile a salient support features to make the tiled support features have a same spatial size as query features, so F can concatenate tiled support features to query features

请问，平铺操作是否是pytorch中的repeat操作呢，然后特征连接是否是pytorch中的cat操作？

Excuse me, is the tiling operation a repeat operation in pytorch, and is the feature connection a cat operation in pytorch?

**Comment by God-song** [comment]
a=torch.randn(1,3,4,4)
m=torch.randn(1,3,4,4)
conv1=nn.AdaptiveAvgPool2d((1,1))
conv2=nn.AdaptiveMaxPool2d((1,1))
print(a)
b=conv1(a)
print(b)
c=conv2(a)
print(c)
d=torch.cat((b,c),dim=1)
print(d)
print(d.shape)
conv3=nn.Conv2d(6,3,1)
e=conv3(d)
print(e)
print(e.shape)
f = e.repeat(1,1,4,4)
print(f)
print(f.shape)
print("b",m.shape)
g=torch.cat((f,m),dim=1)
print(g)
print(g.shape)
